site2/docs/security-jwt.md

```diff
@@ -219,9 +219,21 @@ $ bin/pulsar tokens create-key-pair --output-private-key my-private.key --output
  * Store `my-private.key` in a safe location and only administrator can use `my-private.key` to generate new tokens.
  * `my-public.key` is distributed to all Pulsar brokers. You can publicly share this file without any security concern.
 
-### Generate tokens
 
-A token is the credential associated with a user. The association is done through the "principal" or "role". In the case of JWT tokens, this field is typically referred as **subject**, though they are exactly the same concept.
+### Generate an admin role token
+
+Run the following command to create an admin role token, and use the generated token string as the value of `brokerClientAuthenticationParameters` in the `conf/broker.conf` or `conf/standalone.conf` file.
```

Comment:
We also set up this token to `client.conf`.

Reply:
Updated in a note. PTAL.